packages/site/src/lib/components/Slides.svelte

```diff
@@ -1,35 +1,66 @@
 <script lang="ts">
   export let events: Event[];
+
   import Image from "$lib/components/Image.svelte";
+  import ChevronRight from "@material-symbols/svg-400/rounded/chevron_right.svg?component";
+  import ChevronLeft from "@material-symbols/svg-400/rounded/chevron_left.svg?component";
+
+  interface Poster {
+    altText: string;
+    imageUrl: string;
+  }
 
   interface Event {
-    poster?: {
-      altText: string;
-      imageUrl: string;
-    };
+    poster?: Poster
+  }
+
+  const posters = (): Poster[] => events.flatMap((ev) => ev.poster ? [ev.poster] : [])
+
+  let currentIdx = 0;
+  let current: Poster | undefined = posters()[0]
+
```

Comment:
```suggestion
$: canAdvance = currentIdx < posters.length - 1
$: canGoBack = currentIdx > 0
```

You have two unnamed concepts here, plus you write these twice.